Tasks/DeployVisualStudioTestAgent/TestAgentConfiguration.ps1

```diff
@@ -171,27 +171,28 @@
 
                 $rootFolder = $ScheduleObject.GetFolder('\') #'
                 $newTask = $rootFolder.RegisterTaskDefinition("DTA", $TaskDefinition, 6, '', '', 3)
-                Write-Verbose "Starting scheduled task" -Verbose
-                $rootFolder.DeleteTask("DTA", 0)
+                Write-Verbose "Starting scheduled task on Windows 7." -Verbose
+                Start-Sleep -Seconds 30
```

Comment:
Can we do something better here ..? Loop with 5 seconds interval and break.. Please take it up post this commit.